README.md

```diff
@@ -82,13 +82,13 @@ The current focus of the Bosque project is core language design. As a result the
 
 ### Requirements
 
-#### Windows
-
 In order to build the language for Windows the following are needed:
 
-- The LTS version of [node.js](https://nodejs.org/en/download/) for Windows (x64)
+- 64 bits Operating System

```

Comment:
"bit" or "bits" -- I favor "bit" but either is ok with me

Reply:
```suggestion
- 64 bit Operating System
```

Reply:
bit is ok for me